src/lib.rs

```diff
@@ -535,18 +535,6 @@ pub struct PerConfigContext {
     aggregated_queries: AggregatedQueries<'this>,
 }
 
-impl Clone for PerConfigContext {
```

Comment:
Per previous PR comment, this seemed weird-ish so went ahead and swapped in `maybe_owned::MaybeOwned` for `Cow` (and that doesn't have a `Clone` requirement)